features/step_definitions/landing_page_steps.rb

```diff
@@ -0,0 +1,20 @@
+When(/^I visit the LandingPage$/) do
+  landing_page.load
+end
+
+Then(/^I want to be able to input my age$/) do
+  expect(page).to have_css('#your_details_form_age')
+end
+
+And(/^I want to be able to input my gender$/) do
+  expect(page).to have_css('#your_details_form_gender')
+end
+
+And(/^I want to be able to input my salary$/) do
+  expect(page).to have_css('#your_details_form_salary')
```

Comment:
as @Tr4pSt3R indicates, the feature is merely testing the presence of the fields, not that you can add valid input. That is probably a cucumber spec for a later story. This story merely adds the framework and shows that it works and all the parts have been wired up.